tools/interop_matrix/client_matrix.py

```diff
@@ -264,7 +266,8 @@ def __init__(self, patch=[], runtime_subset=[], testcases_file=None):
         ('v1.15.0', ReleaseInfo(testcases_file='csharp__v1.3.9')),
         ('v1.16.0', ReleaseInfo(testcases_file='csharp__v1.3.9')),
         ('v1.17.1', ReleaseInfo(testcases_file='csharp__v1.3.9')),
-        ('v1.18.0', ReleaseInfo()),
-        ('v1.19.0', ReleaseInfo()),
+        ('v1.18.0', ReleaseInfo(testcases_file='csharpcoreclr__v1.18.0')),
```

Comment:
Several problems:
- This should have been `csharp__v1.18.0` (the distinction between csharp and csharpcoreclr will be handler by the test scripts).
- That file actually doesn't exist (so this PR will break the interop matrix tests)
- Problems would have been revealed by manually starting an adhoc run of interop_matrix tests on this PR (https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod%3Agrpc%2Fcore%2Fexperimental%2Flinux%2Fgrpc_interop_matrix_adhoc)

Reply:
Adding this extra step is good but why is local testing and checking that the images have been uploaded for the new releases not sufficient?

Reply:
Local testing  should be sufficient, but I assume in this case it was only run for v1.20.x (whereas the breakage was introduced for 1.18 and 1.19), or there was just some oversight (e.g. local file not committed).